src/Movie.js

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
 import './Movie.css'
 
-const Movie = ({ id, poster_path, backdrop_path, title, average_rating, release_date }) => {
-  console.log(Math.floor(average_rating))
-
+const Movie = ({ id, poster_path, backdrop_path, title, average_rating, release_date, showProfile }) => {
   const rottenTomatillos = '🤢'
 
+  // const handleClick = (id) => {
+  //   const currentMovie = this.state.allMovies.find(movie => movie.id === id)
+  //   this.setState({ currentMovie: currentMovie })
+  // }
+
+  const showMovie = (id) => {
```

Comment:
Since "showMovie" just calls "showProfile", it's okay to just use "showProfile" in the onClick.